client/elements/sc-page-selector.html

```diff
@@ -145,6 +146,7 @@
         </app-header>
 
         <page-title base-title="" page-title="{{_getPageTitle(toolbarTitle)}}"></page-title>
+        <sherby-metadata data="[[metaData]]"></sherby-metadata>
```

Comment:
This file is a bit different. I think it can stay as-is, although it's not ideal (explained in the next comment).

If you're feeling like improving it, here's how I'd do it:
* make pageTitle a property
* use [[_createMetaData(pageTitle)]] here.

Reply:
This does not work here. The pageTitle needs to be calculated first and cannot be called directly. I already tried that. The only way around would be to use [[_createMetaData(toolbarTitle)]] and then duplicate the function _getPageTitle. Otherwise pageTitle='' when called.

Reply:
if you make `pageTitle` a property, `[[_createMetaData(pageTitle)]]` will get called every time `pageTitle` changes - that's what I meant.

But like I said, it's not essential to have this.